Why: Do not build tools
Source: Ported by Kleis Auke Wolthuizen (wasm-vips): https://github.com/GNOME/glib/compare/2.80.3...kleisauke:wasm-vips-2.80.3 (without "Remove GRegex pre-emptively": Wireshark needs GRegex)

diff --git a/gobject/meson.build b/gobject/meson.build
index 2129aaf8a662687b8031ab3a0faea91f46dea43e..0cf39253dc3d819bc5f1bbe6f20535f4332c4263 100644
--- a/gobject/meson.build
+++ b/gobject/meson.build
@@ -170,11 +170,6 @@ libgobject_dep = declare_dependency(link_with : libgobject,
 )
 meson.override_dependency('gobject-2.0', libgobject_dep)
 
-gobject_query = executable('gobject-query', 'gobject-query.c',
-  install : true,
-  install_tag : 'bin-devel',
-  dependencies : [libglib_dep, libgobject_dep])
-
 install_data('gobject_gdb.py',
   install_dir : glib_pkgdatadir / 'gdb',
   install_tag : 'devel',
